backup_and_restore/application_backup_and_restore/troubleshooting.adoc

```diff
@@ -26,6 +26,7 @@ include::modules/velero-oadp-version-relationship.adoc[leveloffset=+2]
 include::modules/velero-obtaining-by-accessing-binary.adoc[leveloffset=+1]
 
 include::modules/oadp-debugging-oc-cli.adoc[leveloffset=+1]
+// @Wes debugging the Velero resources
```

Comment:
@weshayutin - i could include the oadp-self-signed-certificate offset=+2if that would help